### PR TITLE
TELCODOCS-2056: Post-GA peer review comments for IBU/LCA

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.17, 4.18

Issue: https://issues.redhat.com/browse/TELCODOCS-2056

Link to docs preview:

- https://83133--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/cnf-image-based-upgrade-base.html
- https://83133--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.html
- https://83133--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-auto-image-cleanup.html
- https://83133--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-generate-seed.html
- https://83133--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-install-operators.html
- https://83133--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-prep-resources.html
- https://83133--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-partition.html
- https://83133--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/ztp-image-based-upgrade-prep-resources.html
- https://83133--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/ztp-image-based-upgrade.html

QE review:
- [ ] QE has approved this change.

Additional information: Applying peer review comments from the feature PRs that is relevant to the whole IBU docs.

### CODE REVIEW

File: modules/cnf-image-based-upgrade-troubleshooting.adoc [hunk 1]
```diff
@@ -28,11 +28,11 @@ You can use the `oc adm must-gather` CLI to collect information for debugging an
...
   --dest-dir=must-gather/tmp \
   --image=$(oc -n openshift-lifecycle-agent get deployment.apps/lifecycle-agent-controller-manager -o jsonpath='{.spec.template.spec.containers[?(@.name == "manager")].image}') \
-  --image=quay.io/konveyor/oadp-must-gather:latest \// <1>
-  --image=quay.io/openshift/origin-must-gather:latest <2>
+  --image=quay.io/konveyor/oadp-must-gather:latest \// # <1>
+  --image=quay.io/openshift/origin-must-gather:latest # <2>
 ----
-<1> (Optional) You can add this options if you need to gather more information from the {oadp-short} Operator.
-<2> (Optional) You can add this options if you need to gather more information from the SR-IOV Operator.
+<1> Optional: Add this options if you need to gather more information from the {oadp-short} Operator.
```
Comment (tmalove) [inline modules/cnf-image-based-upgrade-troubleshooting.adoc:34]:
```suggestion
<1> Optional: Add this option if you need to gather more information from the {oadp-short} Operator.
```

File: modules/cnf-image-based-upgrade-troubleshooting.adoc [hunk 2]
```diff
@@ -28,11 +28,11 @@ You can use the `oc adm must-gather` CLI to collect information for debugging an
...
+<2> Optional: Add this options if you need to gather more information from the SR-IOV Operator.
```
Comment (tmalove) [inline modules/cnf-image-based-upgrade-troubleshooting.adoc:35]:
```suggestion
<2> Optional: Add this option if you need to gather more information from the SR-IOV Operator.
```

**Review by tmalove — APPROVED** [review]

**Review by tmalove — APPROVED** [review]

**Review by jeana-redhat — APPROVED** [review]
Excellent clean-up :nerd_face:, LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**